components/src/01-globals/grid/Grid.stories.js

```diff
@@ -9,28 +9,31 @@ storiesOf('Global/Grid', module)
   .add('Default', () => ({
     components: { AlpacaGrid },
     template: `
-        <alpaca-grid
-          :grids="[
-            [
-              'col-xs-12 col-sm-3 col-md-2 col-lg-1',
-              'col-xs-6 col-sm-6 col-md-8 col-lg-10',
-              'col-xs-6 col-sm-3 col-md-2 col-lg-1'
-            ],
-            [
-              'col-xs-12 col-sm-3 col-md-2 col-lg-1',
-              'col-xs-12 col-sm-9 col-md-10 col-lg-11'
-            ],
-            [
-              'col-xs-10 col-sm-6 col-md-8 col-lg-10',
-              'col-xs-2 col-sm-6 col-md-4 col-lg-2'
-            ]
-          ]"
-        />
+       <div class="wrapper">
+          <alpaca-grid
+             :grids="[
+               [
+                 'col-xs-12 col-sm-3 col-md-2 col-lg-1',
+                 'col-xs-6 col-sm-6 col-md-8 col-lg-10',
+                 'col-xs-6 col-sm-3 col-md-2 col-lg-1'
+               ],
+               [
+                 'col-xs-12 col-sm-3 col-md-2 col-lg-1',
+                 'col-xs-12 col-sm-9 col-md-10 col-lg-11'
+               ],
+               [
+                 'col-xs-10 col-sm-6 col-md-8 col-lg-10',
+                 'col-xs-2 col-sm-6 col-md-4 col-lg-2'
+               ]
+             ]"
+          />
+      </div>
     `
   }))
   .add('Fluid', () => ({
     components: { AlpacaGrid },
     template: `
+      <div class="wrapper">
```

Comment:
What do you think to use storybook class `sb-show-main` instead of adding new `wrapper`.